src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -6,9 +6,13 @@
  * @param namespace The name of the namespace.
  * @param collection The name of the collection.
  */
-public record CommandContext(String namespace, String collection) {
+public record CommandContext(String namespace, String collection, boolean isVectorEnabled) {
 
-  private static final CommandContext EMPTY = new CommandContext(null, null);
+  public CommandContext(String namespace, String collection) {
```

Comment:
Do we need to leave this second constructor? I guess it can be convenient for test setup?

Reply:
If I remove this constructor there will be lot of test cases error. If I have to remove this better to do it as separate PR to refactor. Let me know.

Reply:
Ok that's fine let's just leave it.